oshdb-util/src/test/java/org/heigit/bigspatialdata/oshdb/util/geometry/osmtestdata/OSHDBGeometryBuilderTestOsmTestData7xxTest.java

```diff
@@ -962,15 +961,21 @@ public void test776() throws ParseException {
 
   @Test
   public void test777() throws ParseException {
-    // https://gitlab.gistools.geog.uni-heidelberg.de/giscience/big-data/ohsome/oshdb/issues/137
+    /*
+    777 is not really a valid test case.
+
+    According to https://wiki.openstreetmap.org/wiki/Relation:multipolygon: “Generally, the
+    multipolygon relation can be used to build multipolygons in compliance with the OGC Simple
+    Feature standard. Anything that is not a valid multipolygon according to this standard
+    should also be considered an invalid multipolygon relation.”
+
+    Here, the inners form a ring around a portion of the interior.
+    */
+    // https://github.com/GIScience/oshdb/issues/123
     // Multipolygon with two outer rings and two inner rings touching in two nodes
     OSMEntity entity = testData.relations().get(777900L).get(0);
     Geometry result = OSHDBGeometryBuilder.getGeometry(entity, timestamp, tagInterpreter);
-    assertTrue(result instanceof MultiPolygon);
-    assertTrue(result.isValid());
```

Comment:
since the test777 is according to the explanation above not a valid test case, because the touching inner rings are not `valid` in the OGC definition and that's exactly what `isValid` tests. The other asserts below don't make sense because they relied on some "magic" fixing the invalidity of the input data. :shrug: